api/nakadi-event-bus-api.yaml

```diff
@@ -743,6 +743,54 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+  /subscriptions:
+    post:
+      tags:
+        - managed-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: |
+        This endpoint creates a Subscription for EventType(s). The subscription is needed to be able to consume
+        events from EventType(s) in a high level way when Nakadi stores the offsets and manages the rebalancing of
+        consuming clients.
+        The subscription is identified by its parameters. If this endpoint is invoked several times with the same
+        subscription properties in body - the subscription will be created only once and for all calls it will
+        return the same subscription id.
+      parameters:
+        - name: subscription
+          in: body
+          description: Subscription to create
+          schema:
+            $ref: '#/definitions/Subscription'
+          required: true
+      responses:
+        '200':
+          description: Subscription was successfuly created
+          schema:
+            type: string
+            description: id of subscription that was created
+        '401':
+          description: Client is not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '403':
+          description: Client is not authorized to perform this operation
+          schema:
+            $ref: '#/definitions/Problem'
+        '404':
```

Comment:
i guess I though this was under /subscriptions, not /subscriptions/{...}/cursors. 
ignore my comment.